### ISSUE TITLE
aiohttp 2.0 incompatibility

### ISSUE THREAD

**Issue by stephen84s** [elastic/elasticsearch-py-async#15]
Looks like elasticsearch-py-async is incompatible with the 2.0 version of aiohttp.
Reason: recently I started getting the following error:-
```
  File ".../lib/python3.5/site-packages/elasticsearch_async/__init__.py", line 1, in <module>
    from .transport import AsyncTransport
  File ".../lib/python3.5/site-packages/elasticsearch_async/transport.py", line 8, in <module>
    from .connection import AIOHttpConnection
  File ".../lib/python3.5/site-packages/elasticsearch_async/connection.py", line 4, in <module>
    from aiohttp.errors import FingerprintMismatch, ClientError
ImportError: No module named 'aiohttp.errors'
```
Also on checking the master branch I couldn't find a module by the name `aiohttp.errors` however I found it on the `1.3` branch not to mention that forcing the `aiohttp` version down to 1.3 also got rid of the error for me.
So wanted to suggest to fix the `aiohttp` version to `>1.3,<2.0` till elasticsearch_async is migrated to use `aiohttp` 2.0

PS: Still in my early stages of python so feel free to correct me at any point

**Comment by lbx6z** [comment]
aiohttp.errors is gone
https://github.com/aio-libs/aiohttp/issues/1753

**Comment by bmlis** [comment]
Any progress on this? Unfortunately, can't use aiohttp older than 2.0.

**Comment by hellysmile** [comment]
sorry for possible off-topic, but we have started support our own, very similar driver, with some asyncio\aiohttp additional features. 

You are more then welcome to take a look https://github.com/wikibusiness/aioelasticsearch

**Comment by HonzaKral** [comment]
@bmlis I am waiting for the author of the PR to make suggested changes, if I don't hear anything I will address those myself next week and do a release. Sorry about the delays!

**Comment by HonzaKral** [comment]
This has been merged in 4a0f82129bdd8521ea4449b3f73fdba8adcc5a1d Any feedback is welcome!

**Comment by SlavaValAl** [comment]
@HonzaKral hi, are you going to cherry-pick this patch to 2.0.0 version?

**Comment by HonzaKral** [comment]
@SlavaValAl I wasn't planning on it. We just released Elasticsearch `6.0` which makes `2.x` 2 major versions behind. If there is significant need, I am open to it, however.

**Comment by SlavaValAl** [comment]
@HonzaKral, unfortunately, we have huge amount of elastic 2.0 instances, and, I'm not sure when we are going to switch to actual version.
